lib/screens/home.dart

```diff
@@ -1,13 +1,29 @@
+import 'package:curimba/enums/view_state.dart';
+import 'package:curimba/helpers/notifications_helper.dart';
 import 'package:curimba/helpers/shared_preferences_helper.dart';
+import 'package:curimba/view_models/home_view_model.dart';
 import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
 
 import '../utils/locator.dart';
 import '../utils/navigation_service.dart';
+import 'base_view.dart';
 
-class Home extends StatelessWidget {
+class Home extends StatefulWidget {
   @override
+  HomeState createState() => HomeState();
+}
+
+class HomeState extends State<Home> {
```

Comment:
Fazer um `StatefulWidget` com o `BaseView` é meio que redundante, o `BaseView` cria e retorna já um `StatefulWidget`, recomendo manter como `StatelessWidget`
```suggestion
class Home extends StatelessWidget {
```

Reply:
é que aqui eu precisei fazer assim para mudar o ícone de notificações, acho que não da com stateless widget